src/scene.ts

```diff
@@ -4480,6 +4480,14 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         this.afterRender = null;
         this.metadata = null;
 
+        if (EngineStore._LastCreatedScene === this) {

```

Comment:
OVerall I'm good with this change but you need to remove the next block of code (duplicate)

Reply:
I already did it before you comment ;)